### PR TITLE
Removes some old and broken features

### PR DESCRIPTION
**Pull request by AffectedArc07**
## What Does This PR Do
This PR removes the following oldcode features:
- Atmos Automations
- Logic Gates
- Trams

Each one being justified below:

Atmospherics Automation was put in by a coder who has not been able to contribute here for over 5 years, and the system has not been *touched* since. The code barely works, its never used (I can recall like 4 instances of it being used in my time here [4 years]). It is barely supported by atmos machines, requires hacky code, and is not worth keeping ingame in its broken state. Hell, the console screen doesnt even have a sprite, its just black

Logic Gates only have signalers as inputs and lightswitches as outputs. Couple that with the fact they are slow as hell and also a DOS vector, and they are full-tile machines instead of items, I really see no use for these, or a reason for them to exist

Trams are ancient oldcode. 99% of people arent even aware they exist, and they have not been used at all, dont work anymore, and were only added as an early development project.

This PR also removes an unticked file (tools.dm) as well as fixing 2 minor code issues which the langserver complained at.

## Why It's Good For The Game
Stuff which barely works and brings the general quality of the code down shouldnt be here

## Changelog
:cl:
fix: Fixes a few problems the langserver complained at
del: Removed Atmospheric Automations Console
del: Removed logic gates
del: Removed Trams (Yes, trams. We had those)
/:cl:

### CODE REVIEW

**Comment by taukausanake** [comment]
Awwww, I wanted to mess with the trams. They sounded nice for an away mission like for a lab or something. You sure they don't work anymore and not just tossing them aside because oldcode?

**Comment by AffectedArc07** [comment]
I have been told they dont even work, but youre welcome to test. We dont want them here, but youre welcome to revert them in your `para-revert` branch.

**Comment by Mitchs98** [comment]
Is the atmos automation console the one in Atmos that lets you fiddle with the air alarms without leaving atmos?

**Comment by AffectedArc07** [comment]
@Mitchs98 no, that’s the central atmospherics computer. The atmos automation computer is printed in R&D and allows “if this then that” control of atmos devices. It’s broken as shit and doesn’t even have an on texture.

File: code/modules/pda/app.dm [hunk 1]
```diff
@@ -50,8 +50,6 @@
 		if(!pda.notifying_programs.len)
 			pda.overlays -= image('icons/obj/pda.dmi', "pda-r")
 
-/datum/data/pda/proc/
```
Comment (Fox-McCloud) [inline code/modules/pda/app.dm:53]: whut.

**Comment by Mitchs98** [comment]
> @Mitchs98 no, that’s the central atmospherics computer. The atmos automation computer is printed in R&D and allows “if this then that” control of atmos devices. It’s broken as shit and doesn’t even have an on texture.

Ah okay. I only atmos like maybe one in twenty rounds so I barely know the machinery, didn't think it was but I wanted to be sure. Don't even know what that is/it existed.

File: code/modules/pda/app.dm (hunk 1 above)
Comment (AffectedArc07) [reply code/modules/pda/app.dm:53]: Sometime I sit and ask myself "How did this even compile"

**Pull request merged**